prbench/src/prbench/envs/dynamic3d/robots/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""Robot classes for dynamic3d environments."""
+
+from prbench.envs.dynamic3d.robots.base import RobotEnv  # pylint: disable=unused-import
```

Comment:
to avoid all these unused imports and make it clear what's going on in this file, you can do

```python
__all__ = [RobotEnv, RBY1ARobotActionSpace, RBY1ARobotEnv, TidyBot3DRobotActionSpace, TidyBotRobotEnv]
```